test/meetup.spec.js

```diff
@@ -5,24 +5,15 @@ import { expect } from 'chai';
 
 import server from '../index';
 
-describe('Meetups List Api Exists', () => {
+describe('Meetups', () => {
   describe('GET /meetups', () => {
-    it('should return status code 200 on successful fetching', (done) => {
+    it('should return status code 200 on successful fetching of all meetups', (done) => {
       request(server)
         .get('/api/v1/meetups')
         .end((err, res) => {
           expect(res.statusCode).to.equal(200);
-          done();
-        });
-    });
-  });
-
-  describe('GET /meetups/upcoming', () => {
-    it('should return status code 200 for successful fetching of upcoming meetups', (done) => {
-      request(server)
-        .get('/api/v1/meetups/upcoming')
-        .end((err, res) => {
-          expect(res.statusCode).to.equal(200);
+          expect(res.body).to.be.an('object');
+          expect(res.body).to.have.property('data');
```

Comment:
You didn't change it here @McHardex

Reply:
Please take some time to review your code before pushing

Reply:
Is it OK to just check if the keys inside data object exists?